index.qmd

```diff
@@ -5,19 +5,19 @@ subtitle: "Resources for the US GHG Center"
 
 ## Welcome {#welcome}
 
-The [U.S. Greenhouse Gas (GHG) Center](https://ghg.center) provides a cloud-based system for exploring and analyzing U.S. government and other curated greenhouse gas datasets.
+The [U.S. Greenhouse Gas (GHG) Center](https://ghg.center/) provides a cloud-based system for exploring and analyzing U.S. government and other curated greenhouse gas datasets.
 
-On this site, you can find the technical documentation of the services the center provides, how to load the datasets, and how the datasets were transformed from their source formats (eg. NetCDF, HDF, etc.) into cloud-optimized formats that enable efficient data access and visualization.
+On this site, you can find the technical documentation for the services the center provides, how to load the datasets, and how the datasets were transformed from their source formats (eg. netCDF, HDF, etc.) into cloud-optimized formats that enable efficient cloud data access and visualization.
 
 
 ## Contents
 
-1. **Services** provided for accessing and analyzing the GHG Center datasets, such as a [JupyterHub](services/jupyterhub.qmd) environment for interactive computing.
-2. Dataset **usage** examples, e.g. for the [LPJ-wsl modelled Wetland Methane Emissions](user_data_notebooks/lpjwsl-wetlandch4-grid-v1_User_Notebook.ipynb) dataset, showing how to load the dataset in Python, for example in [JupyterHub](services/jupyterhub.qmd).
-3. Dataset **transformation** scripts, e.g. for the [CASA-GFED3 Land Carbon Flux](cog_transformation/casagfed-carbonflux-monthgrid-v3.ipynb) dataset.
-4. Data **processing and verification** reports, e.g. for the [CEOS CH₄ budget yearly](processing_and_verification_reports/gosat-based-ch4budget-yeargrid-v1_Processing and Verification Report.qmd) dataset.
+1. Services provided for accessing and analyzing the US GHG Center datasets, such as the [JupyterHub](services/jupyterhub.qmd) environment for interactive computing.
+2. Dataset usage examples, e.g. for the [Wetland Methane Emissions from the LPJ-wsl model](user_data_notebooks/lpjwsl-wetlandch4-grid-v1_User_Notebook.ipynb) dataset, that shows how to load the dataset in Python in [JupyterHub](services/jupyterhub.qmd).
+3. Dataset transformation scripts, such as the [CASA-GFED3 Land Carbon Flux](cog_transformation/casagfed-carbonflux-monthgrid-v3.ipynb) dataset.
+4. Data processing and verification reports that openly present the process we used to check and verify that any transformation did not alter the original source data. An example is the [CEOS CH₄ budget yearly](processing_and_verification_reports/gosat-based-ch4budget-yeargrid-v1_Processing and Verification Report.qmd) dataset.
```

Comment:
can you keep the bold words like before?

Reply:
It seemed different from the content that deborah provided that's why I changed it.